sdk/storage/azure-storage-stress/src/main/java/com/azure/storage/stress/ContentInfo.java

```diff
@@ -0,0 +1,54 @@
+package com.azure.storage.stress;
+
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.nio.ByteBuffer;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.zip.CRC32;
+
+public class ContentInfo {
+    private final long length;
+    private final long crc;
+    private final ByteBuffer head;
+
+    public ContentInfo(long crc, long length, ByteBuffer head) {
+        this.crc = crc;
+        this.length = length;
+        this.head = head;
+    }
+
+    public static Mono<ContentInfo> fromFluxByteBuffer(Flux<ByteBuffer> data) {
+        AtomicLong length = new AtomicLong(0);
+        ByteBuffer head = ByteBuffer.allocate(1024);
```

Comment:
nit: Not certain about the performance requirements of the stress tests but using `get()` and `put()` is inefficient as that is a byte-by-byte copy. Since the `head` ByteBuffer uses allocate it will have a backing `byte[]`, with some more logic we should be able to hook into `get(byte[], int, int)` to retrieve chunks of bytes in one go.